src/Home/Project.js

```diff
@@ -40,20 +39,20 @@ const OpenSourceStat = styled(({
   indicateColor,
   label,
   dataAgeInDays,
-  className
+  className,
 }) => {
   const disabled = typeof value !== 'number'
   const changeValue = parseFloat(change, 10) > 0 ? `+${change}` : change
 
   return (
     <div title={label} className={`${className} ${disabled ? 'disabled' : ''}`}>
       <OpenSourceStatIcon>
-        <Icon/>
+        <Icon />
       </OpenSourceStatIcon>
-      {disabled ? <div>N/A</div> :
+      {disabled ? <div>N/A</div> : (
         <div>
           <strong>{value}</strong>
-          {dataAgeInDays < 1 ? null :
+          {dataAgeInDays >= 1 &&
             <OpenSourceStatChange
               title={`${label} in the last ${dataAgeInDays} days`}
```

Comment:
**TODO:** Should be `day` instead of `days` when `dataAgeInDays === 1`.

Reply:
Yeah just didn't bother with it in the initial implementation, but the improvement is more than welcome. Makes sense to do in a separate PR.

Reply:
https://github.com/netlify/headlesscms.org/pull/116